client-flink/flink-1.14/src/test/java/org/apache/celeborn/plugin/flink/ShuffleTaskInfoSuitJ.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.celeborn.plugin.flink;
+
+import org.junit.Test;
+
+public class ShuffleTaskInfoSuitJ {
+  @Test
+  public void testEncode() {
+    ShuffleTaskInfo shuffleTaskInfo = new ShuffleTaskInfo();
+    int encodeShuffleId = shuffleTaskInfo.getShuffleId("shuffleId");
+    assert (encodeShuffleId == 0);
+
+    // another shuffle
+    int encodeShuffleId1 = shuffleTaskInfo.getShuffleId("shuffleId1");
+    assert (encodeShuffleId1 == 1);
```

Comment:
Do you mean org.junit.Assert? IMO we need to replace java assert to org.junit.Assert in test.

Reply:
@RexXiong modified, please take a look